reporter/sbom/spdx.go

```diff
@@ -0,0 +1,540 @@
+package sbom
+
+import (
+	"cmp"
+	"encoding/json"
+	"fmt"
+	"math/rand/v2"
+	"slices"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/spdx/tools-golang/spdx"
+	"github.com/spdx/tools-golang/spdx/v2/common"
+
+	"github.com/future-architect/vuls/constant"
+	"github.com/future-architect/vuls/models"
+)
+
+const (
+	creatorOrganization = "future-architect"
+	creatorTool         = "vuls"
+	documentNamespace   = "https://www.future.co.jp"
```

Comment:
Feels that it would be better to add a path element like `/spdxdoc` as described https://www.hitachi-solutions.co.jp/sbom/blog/2021120102/ (sorry, it's in Japanese).